src/components/Featured/index.js

```diff
@@ -8,74 +8,90 @@ import styles from "./styles.module.css";
 
 const features = [
   {
-    title: <>Threat Intelligence and AI</>,
-    imageUrl: "/img/Cortex_Logo.svg",
-    toPage: "https://cortex.pan.dev",
-    color: "cortex",
+    title: <>Cloud Security</>,
+    imageUrl: "/img/PrismaCloud.png",
+    toPage: "https://prisma.pan.dev",
+    target: "_self",
+    color: "prisma",
     description: (
       <>
         <p className="text text--secondary">
-          Get started with developing solutions with Cortex XDR and Cortex Data
-          Lake. Find API reference docs, tutorials and more.
+          Discover the APIs, tools and techniques necessary for bringing DevOps
+          practices to the cloud.
         </p>
       </>
     ),
   },
   {
-    title: <>Cloud Automation and Security</>,
-    imageUrl: "/img/Prisma_Logo.svg",
+    title: <>Secure Access Service Edge</>,
+    imageUrl: "/img/PrismaSASE.png",
```

Comment:
`imageUrl` may be case-sensitive?

![image](https://user-images.githubusercontent.com/48506502/162536134-a9b6a739-6cd7-488f-803f-5f534d035d17.png)